src/App.js

```diff
@@ -13,31 +13,12 @@ class App extends Component {
     super();
     this.state = {
       movies: [],
-      selectedMovie: 0,
+      selectedMovie: null,
```

Comment:
Can the initial value for this be an empty object instead of null? Always best to avoid mutating the data type.